test/browser/support/setup.js

```diff
@@ -0,0 +1,27 @@
+const { Before, BeforeAll, AfterAll, After } = require("@cucumber/cucumber")
+const { chromium } = require("playwright")
+
+BeforeAll(async function () {
+  global.browser = await chromium.launch({
+    // Not headless so we can watch test runs
+    headless: true,
+    // Slow so we can see things happening
+    // slowMo: 500
```

Comment:
Is this so we can uncomment later if we want to slow this down?

Reply:
It's the default code I took from mobile id check repo. So I think it's not bad to keep the comment.
We don't have to uncomment it because it's useful when you have to wait/test backend response. And demo app doesn't has any backend, and doesn't wait any answer from GTM.

Reply:
Makes sense thank you!